search-app/src/cmr/search/services/community_usage_metrics/metrics_service.clj

```diff
@@ -30,15 +33,70 @@
   (when (>= col 0)
     (nth csv-line col)))
 
+(defn- get-short-name-by-entry-title
+  "Query elastic for a collection with a given entry-title, returns short-name"
+  [context entry-title]
+  (when (seq entry-title)
+    (let [condition (qm/string-condition :entry-title (str entry-title "*") true true)
```

Comment:
why do we search with pattern here? Also I don't think we want to search with case sensitive.

Reply:
good catch, I thought it was insensitive.  I used a pattern because I think they use product id which is entry-title+ some version number.